src/components/ebay-select/select.stories.ts

```diff
@@ -138,7 +138,7 @@ ExternalLabel.parameters = {
 };
 
 ExternalLabel.args = {
-    options: [
+    option: [ //This breaks if I set it to option, but it looks like everything's already swamped.
```

Comment:
Is this something you meant to fix before merging? I'm not sure I understand the comment